jetbrains-core/src/software/aws/toolkits/jetbrains/services/clouddebug/actions/PseCliAction.kt

```diff
@@ -44,22 +43,17 @@ import software.aws.toolkits.jetbrains.services.ecs.EcsClusterNode
 import software.aws.toolkits.jetbrains.services.ecs.EcsServiceNode
 import software.aws.toolkits.jetbrains.services.ecs.resources.EcsResources.describeService
 import software.aws.toolkits.jetbrains.services.ecs.resources.EcsResources.listServiceArns
-import software.aws.toolkits.jetbrains.services.telemetry.TelemetryConstants.CLOUDDEBUG_TELEMETRY_PREFIX
-import software.aws.toolkits.jetbrains.services.telemetry.TelemetryConstants.CLOUDDEBUG_VERSION
-import software.aws.toolkits.jetbrains.services.telemetry.TelemetryConstants.RESULT
-import software.aws.toolkits.jetbrains.services.telemetry.TelemetryConstants.TelemetryResult
-import software.aws.toolkits.jetbrains.services.telemetry.TelemetryService
 import software.aws.toolkits.jetbrains.utils.notifyError
 import software.aws.toolkits.jetbrains.utils.notifyInfo
 import software.aws.toolkits.resources.message
-import java.time.Duration
+import software.aws.toolkits.telemetry.Result
```

Comment:
I keep wondering what this gains instead of plain old true/false :)

Reply:
there's actually a triplicate - Success, Failure, Canceled


But I agree - I think there should be an override that allows a "Success/Failure" boolean expression.

Reply:
Ask and ye shall receive: https://github.com/aws/aws-toolkit-common/pull/22 . I wanted to add this as well, will update this PR with true/false vs Result where appropriate.